CONTRIBUTING.md

```diff
@@ -74,6 +77,22 @@ Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-
 - Dates in a release are approximations and always subject to change.
 - `Next` milestone is for work that has been triaged, but not prioritized/accepted for any release.
 
+## Proposal process (CAEP)
+
+The Cluster API Enhacement Proposal is the process this project uses to adopt new features, or changes to the APIs.
+
+The [template](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/proposals/YYYYMMDD-template.md), and accepted proposals live under [docs/proposals](https://github.com/kubernetes-sigs/cluster-api/tree/master/docs/proposals).
+
+- Proposals or requests for enhacements (RFEs) MUST be discussed in an issue first.
```

Comment:
Can this be reworded to something along the lines of: Proposals MUST have an associated issue? That way it doesn't sound like we are gate keeping folks from trying to write a proposal if they feel that it would more clearly state their case than doing so in an issue directly.

That way it is clear that the issue is required for being included in a release rather than being a prerequisite step for someone starting to put together a proposal if they want to.

Reply:
Good feedback! Thanks, reworded